src/render.ts

```diff
@@ -146,17 +150,31 @@ export async function renderPlan({
   planfile: string
   terraformCommand: string
   workingDirectory: string
-}): Promise<RenderedPlan> {
+}): Promise<Array<RenderedPlan>> {
   const options = {
     cwd: workingDirectory,
     silent: true
   }
-  const structuredPlanfile = await exec
-    .getExecOutput(terraformCommand, ['show', '-json', planfile], options)
-    .then((output) => JSON.parse(output.stdout))
-    .then((json) => parsePlanfileJSON(json))
   const humanReadablePlanfile = await exec
     .getExecOutput(terraformCommand, ['show', '-no-color', planfile], options)
     .then((output) => output.stdout)
-  return internalRenderPlan(structuredPlanfile, humanReadablePlanfile)
+  try {
+    const structuredPlanfile = await exec
+      .getExecOutput(terraformCommand, ['show', '-json', planfile], options)
+      .then((output) => JSON.parse(output.stdout))
+      .then((json) => parsePlanfileJSON(json))
+    return [internalRenderPlan(structuredPlanfile, humanReadablePlanfile)]
+  } catch (error) {
+    if (error instanceof SyntaxError) {
+      const jsonPlans = await exec
+        .getExecOutput(terraformCommand, ['show', '-json', planfile], options)
+        .then((output) => output.stdout.split('\n'))
+        .then((plans) => plans.filter((plan) => plan !== ''))
+      return jsonPlans
+        .map((plan) => JSON.parse(plan))
+        .map((json) => parsePlanfileJSON(json))
+        .map((structuredPlanfile) => internalRenderPlan(structuredPlanfile, humanReadablePlanfile))
+    }
```

Comment:
Correct me if I'm wrong but I believe that this suffers from the issue that we don't ever "merge" the different resource changes? 🤔 i.e. you'll get separate created/changed/deleted blocks for each individual resource, right?